packages/react-error-overlay/src/containers/StackFrame.js

```diff
@@ -174,7 +174,9 @@ class StackFrame extends Component {
               onClick={canOpenInEditor ? this.openInEditor : null}
               style={canOpenInEditor ? codeAnchorStyle : null}
             >
-              <CodeBlock {...codeBlockProps} />
+              {// Use JS instead of JSX spread attributes to avoid Object.assign
```

Comment:
That's unfortunate. Are you sure this is really an issue? I had an impression that JSX puts extends helper in either case.

Reply:
Yes. We have enabled `useBuiltIns` flag for `babel-plugin-transform-react-jsx` as well.
https://github.com/facebookincubator/create-react-app/blob/master/packages/babel-preset-react-app/index.js#L27